en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md

```diff
@@ -84,30 +83,6 @@ WSO2 API Manager that can be used for logging.
 
 In order to enable logging by invoking `APILogMessageHandler` , follow the steps below.
 
-**To enable Message Logging per API :**
-
-1.  Open the synapse configuration of the API located in `<APIM_HOME>/repository/deployment/server/synapse-configs/default/api` 
-directory and add below handler before `</Handlers>` .
-
-    ``` java
-    <handler class="org.wso2.carbon.apimgt.gateway.handlers.logging.APILogMessageHandler"/> 
-    ```
-
-2.  Copy the following code into the `<APIM_HOME>/repository/conf/log4j2.properties` file to enable printing DEBUG logs.
-
-    ``` java
-    logger.log-msg-handler.name = org.wso2.carbon.apimgt.gateway.handlers.logging.APILogMessageHandler
-    logger.log-msg-handler.name = DEBUG
-    ```
-    Append the `log-msg-handler` logger name to `loggers` configuration which is a comma separated list of all active loggers. Sample configuration can be seen below.
-
-    ```
-    loggers = log-msg-handler, trace-messages, org-apache-coyote,com-hazelcast
-    ```
-
-    !!! note
-        The logger name `log-msg-handler` can be replaced by any logger-name.
-
```

Comment:
I think we still need this part which enables the debug log level. Otherwise the logs will not be printed.. is it possible to correct this? @isharac

Reply:
Please include this change too https://github.com/wso2/docs-apim/pull/3802

Reply:
added with https://github.com/wso2/docs-apim/pull/3803